src/test/java/uk/gov/digital/ho/hocs/workflow/api/dto/MigrateCaseResponseTest.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.digital.ho.hocs.workflow.api.dto;
+
+import org.junit.Test;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MigrateCaseResponseTest {
+
+    @Test
+    public void getMigrateCaseResponse() {
```

Comment:
I'm not sure why you're bothering with this test, it looks like you're testing whether a java constructor works... we know they do, there's no extra logic in the constructor. I'd recommend removing it.